tests/test_thunderbird_urls.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+import pytest
+import requests
+
+
+@pytest.mark.skip_selenium
+@pytest.mark.nondestructive
+class TestThunerbirdURLs(object):
+
+    paths = [
+        '/thunderbird/all/',
+        '/thunderbird/releases/',
+    ]
+
+    def test_urls_ok(self, mozwebqa):
+        for path in self.paths:
+            url = mozwebqa.base_url + '/en-US' + path
+            response = requests.get(url, timeout=3)
+            assert len(response.history) <= 2
```

Comment:
There's code readability and failure readability to take into account. I'm not against moving away from unittestzero.